css/oembed.css

```diff
@@ -0,0 +1,3 @@
+.oembed-required {
+    color: #B40000 !important;

```

Comment:
Optional: Maybe apply the same hex code (#f00) that Drupal uses (for consistency)?

Reply:
That colour code is the one we have defined in the sass variables file.

Reply:
It seems we uses two shades of red then.  I was looking at the "backend" when editing a node, which uses the Drupal's theme system .form-required #f00.   The #b40000 is used in webform which we customized in our sass file.

![image](https://user-images.githubusercontent.com/34037404/36788632-aac19dca-1c5c-11e8-89f7-66026160b935.png)